Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ExternalDB/ExternalDBInfo.tsx

```diff
@@ -64,7 +61,7 @@ export default function ExternalDBInfo(props: { Record: SystemCenter.Types.Exter
                 </div>
             </div>
             <div className="card-body" style={{ height: window.innerHeight - 350, maxHeight: window.innerHeight - 350 }}>
```

Comment:
setting `height` to a `px` value is bad layout. Let's fix this all the way up to be a `flex` row